File: pkg/codegen/testing/test/testdata/transpiled_examples/getting-started-pp/nodejs/getting-started.ts
import * as pulumi from "@pulumi/pulumi";
import * as aws from "@pulumi/aws";

const mybucket = new aws.s3.Bucket("mybucket", {website: {
    indexDocument: "index.html",
}});
const indexhtml = new aws.s3.BucketObject("indexhtml", {
    bucket: mybucket.id,
    source: new pulumi.asset.StringAsset("<h1>Hello, world!</h1>"),
    acl: "public-read",
    contentType: "text/html",
});
export const bucketEndpoint = pulumi.interpolate`http://${mybucket.websiteEndpoint}`;

File: tests/testdata/codegen/transpiled_examples/getting-started-pp/go/getting-started.go
package main

import (
	"fmt"

	"github.com/pulumi/pulumi-aws/sdk/v5/go/aws/s3"
	"github.com/pulumi/pulumi/sdk/v3/go/pulumi"
)

func main() {
	pulumi.Run(func(ctx *pulumi.Context) error {
		mybucket, err := s3.NewBucket(ctx, "mybucket", &s3.BucketArgs{
			Website: &*s3.BucketWebsiteArgs{
				IndexDocument: "index.html",
			},
		})
		if err != nil {
			return err
		}
		_, err = s3.NewBucketObject(ctx, "indexhtml", &s3.BucketObjectArgs{
			Bucket:      mybucket.ID(),
			Source:      pulumi.NewStringAsset("<h1>Hello, world!</h1>"),
			Acl:         "public-read",
			ContentType: "text/html",
		})
		if err != nil {
			return err
		}
		ctx.Export("bucketEndpoint", mybucket.WebsiteEndpoint.ApplyT(func(websiteEndpoint string) (string, error) {
			return fmt.Sprintf("http://%v", websiteEndpoint), nil
		}).(pulumi.StringOutput))
		return nil
	})
}
